README.md

```diff
@@ -76,3 +95,82 @@ npm test | tap-color
 ```
 
 ![Colorized output](docs/tap-color-screenshot.png)
+
+
+## API
+
+### describe
+
+```js
+describe = (unit: String, cb: TestFunction) => Void
+```
+
+Describe takes a prose description of the unit under test (function, module, whatever), and a callback function (`cb: TestFunction`). The callback function should be an [async function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function) so that the test can automatically complete when it reaches the end. RITEWay assumes that all tests are asynchronous. Async functions automatically return a promise in JavaScript, so RITEWay knows when to end each test.
+
+
+### TestFunction
+
+```js
+TestFunction = assert => Promise | Void
+```
+
+The `TestFunction` is a user-defined function which takes `assert()` and should return a promise. If you make your `TestFunction` async, it will return a promise automatically. If you don't make your function async, you can destructure `end()` from `assert()` and call it when you're done to end the test. For example, this is the actual unit test used to test the `end()` function (yes, we used RITEWay to test RITEWay):
+
+```js
+describe('synchronous assert()', ({assert, end}) => {
+
+  assert({
```

Comment:
This test case doesn't really work well for the documentation like the other test cases do.

Reply:
Can you think of a better one?

Reply:
Maybe just use sum or something simple again. The reason why it seems odd is that there was no actual/expected value in the assertion

Reply:
Ah, well. That's a neat little trick that's basically equivalent to `pass()` except you still have to provide more detail with `given` and `should`. It's not really that there's no `actual` and `expected` -- they're just `undefined`. ;)

Reply:
Maybe we should just ditch `end()` entirely.

Reply:
Ahh ok cool :D